src/helpers/WLClasses.cpp

```diff
@@ -8,16 +8,23 @@ SLayerSurface::SLayerSurface() {
 }
 
 void SLayerSurface::applyRules() {
-    noAnimations = false;
-    forceBlur    = false;
-    ignoreZero   = false;
+    noAnimations     = false;
+    forceBlur        = false;
+    ignoreAlpha      = false;
+    ignoreAlphaValue = 0.f;
 
     for (auto& rule : g_pConfigManager->getMatchingRules(this)) {
         if (rule.rule == "noanim")
             noAnimations = true;
         else if (rule.rule == "blur")
             forceBlur = true;
-        else if (rule.rule == "ignorezero")
-            ignoreZero = true;
+        else if (rule.rule.find("ignorealpha") == 0 || rule.rule.find("ignorezero") == 0) {
+            const std::string VALUE = removeBeginEndSpacesTabs(rule.rule.substr(11));
+            try {
+                ignoreAlpha = true;
+                if (VALUE.size() != 0)
+                    ignoreAlphaValue = std::stof(VALUE);
+            } catch (...) {}
```

Comment:
`Debug::log(ERR, "Invalid value passed to ignoreAlpha");`

Reply:
https://github.com/hyprwm/Hyprland/pull/2477/commits/af5004f79b99f109f6f68275590f7961e5d7776d